cukinia/common_security_tests.d/sysctl.conf

```diff
@@ -37,3 +37,18 @@ as "SEAPATH-00172 - Check kexec is disabled" cukinia_test \
 
 as "SEAPATH-00173 - Check binfmt_misc is disabled" cukinia_test \
     "$(sysctl fs.binfmt_misc.status)" == "fs.binfmt_misc.status = disabled"
+
+as "ANSSI-BP-028-R14-1 - Check config filesystem" cukinia_test \
+    	"$(sysctl fs.suid_dumpable)" == "fs.suid_dumpable = 0"
+
+as "ANSSI-BP-028-R14-2 - Check config filesystem" cukinia_test \
```

Comment:
Use different test descriptions. Ex:
 1. Check suid_dumpable is disabled
 2. Check protected_fifos is enabled
 3. ...